data.json

```diff
@@ -1546,5 +1546,11 @@
     "githubUsername": "mayankchaudhary26",
     "imageUrl": "https://user-images.githubusercontent.com/56837137/139258468-f7b162b5-94f2-4fb3-94e0-7da43db795dd.png",
     "issueNumber": 805
-  }
+  },
+  {
+  "name": "Antonio Martínez",
+  "githubUsername": "metantonio",
+  "imageUrl": "https://user-images.githubusercontent.com/13443750/139855574-e458b36b-2e02-4d6e-86f7-8c33831fc94a.jpg",
+  "issueNumber": 820
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/139906982-fb25c6da-4255-41b6-9f53-71f48f78d32b.png)

```suggestion
      "name": "Antonio Martínez",
      "githubUsername": "metantonio",
      "imageUrl": "https://user-images.githubusercontent.com/51391473/139906982-fb25c6da-4255-41b6-9f53-71f48f78d32b.png",
      "issueNumber": 820
```